python/src/CrossEntropyImportanceSampling_doc.i.in

```diff
@@ -45,7 +111,17 @@ quantileLevel : : float
 
 Returns
 -------
-CrossEntropyResult : : class:`~openturns.CrossEntropyResult`
+CrossEntropyResult : :class:`~openturns.experimental.CrossEntropyResult`
```

Comment:
```suggestion
CrossEntropyResult : :class:`~openturns.CrossEntropyResult`
```

This class will not be experimental forever and I am afraid we will forget to change this back when the time comes. As a result of this change, the link will be broken but this is an acceptable tradeoff IMO.

Reply:
If we don't put the "experimental", the link is not active (this is the cause of the formatting error you mentioned earlier).

Reply:
maybe we could discuss that point during the next technical committee as we should adopt the same convention for all the experimental classes.

Reply:
Following the TC decision, keeping "experimental" in the link is OK, and we also need to automatically detect broken links.